src/Behat/Testwork/Argument/MixedArgumentOrganiser.php

```diff
@@ -112,26 +112,26 @@ private function isStringKeyAndExistsInParameters($argumentKey, $parameterNames)
     }
 
     /**
-     * Tries to find a parameter number, which typehints provided value.
+     * Check if a given value is typehinted in the argument list.
      *
-     * @param ReflectionParameter[] $parameters
-     * @param mixed                 $value
+     * @param  ReflectionParameter[] $parameters
+     * @param  mixed                 $value
      *
-     * @return null|integer
+     * @return Boolean
      */
-    private function getParameterNumberWithTypehintingValue(array $parameters, $value)
+    public function isParameterTypehintedInArgumentList(array $parameters, $value)
```

Comment:
Ah, it was a new method that I added in, automatically went as public. I had forgotten to check the visibility of other methods. I have updated this one.